### ISSUE TITLE
BUG: No named parameter 'nullOk'

### ISSUE THREAD

**Issue by priyamharsh14** [iampawan/VelocityX#72]
**Describe the bug**
```
/C:/development/flutter/.pub-cache/hosted/pub.dartlang.org/velocity_x-2.3.1-nullsafety.0/lib/src/extensions/context_ext.dart:177:36: Error: No named parameter with the name 'nullOk'.
      Actions.invoke(this, intent, nullOk: nullOk) as bool?;
                                   ^^^^^^
/C:/development/flutter/packages/flutter/lib/src/widgets/actions.dart:898:18: Context: Found this candidate, but the arguments don't match.
  static Object? invoke<T extends Intent>(
                 ^^^^^^
                                                                        
                                                                        
FAILURE: Build failed with an exception.
```
I tried to run a flutter project, but I ended up with this error.
I am also attaching my dependencies list
```
cupertino_icons: ^1.0.2
  http: ^0.13.0
  package_info: ^2.0.0
  launch_review: ^2.0.0
  url_launcher: ^6.0.2
  firebase_core: ^1.0.0
  firebase_auth: ^1.0.0
  firebase_messaging: ^9.0.0
  fluttertoast: ^7.0.4
  velocity_x: ^2.3.1-nullsafety.0
  firebase_auth_web: ^1.0.0
  pin_code_fields: ^6.1.0
  shared_preferences: ^2.0.3
  simple_animations: ^2.2.2
  supercharged: ^1.10.0
  keyboard_avoider: ^0.1.2
  intl: ^0.17.0
```

**Comment by laiiihz** [comment]
it has been repaired on branch dev#null-safety [#71](https://github.com/iampawan/VelocityX/pull/71#issue-585330350)

**Comment by priyamharsh14** [comment]
@laiiihz but how can I use that branch in my flutter project?

**Comment by laiiihz** [comment]
something like this should work @priyamharsh14 
```yaml
  velocity_x:
    git:
      url: https://github.com/iampawan/VelocityX
      ref: dev#null-safety
```

**Comment by iampawan** [comment]
I will push the new update soon but for now, either use the above suggestion by @laiiihz or use v1.4.1

**Comment by priyamharsh14** [comment]
v1.4.1 is creating the #70 error, that's why I can't use it right now.
For now, I will use the above workaround.

Thanks

**Comment by iampawan** [comment]
Fixed in version - ^2.4.0-nullsafety.0
